app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/codePager/SampleInfoScreen.kt

```diff
@@ -199,31 +185,49 @@ fun CodeViewerFile(title: String, iconId: Int) {
 }
 
 @Composable
-@OptIn(ExperimentalMaterial3Api::class)
-private fun CodePagerTopAppBar(
-    scrollBehavior: TopAppBarScrollBehavior,
-    onBackPressed: () -> Unit,
-    title: String
+private fun CodePageView(
+    codePagerTitles: List<String>,
+    sample: Sample,
+    optionPosition: Int
 ) {
-    TopAppBar(
-        title = {
-            Text(
-                text = title,
-                color = MaterialTheme.colorScheme.onPrimary
-            )
-        },
-        colors = TopAppBarDefaults.topAppBarColors(
-            containerColor = MaterialTheme.colorScheme.primary,
-        ),
-        scrollBehavior = scrollBehavior,
-        navigationIcon = {
-            IconButton(onClick = onBackPressed) {
-                Icon(
-                    imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
-                    contentDescription = LocalContext.current.getString(R.string.backButton),
-                    tint = MaterialTheme.colorScheme.onPrimary
+    var selectedFileIndex by remember { mutableIntStateOf(optionPosition) }
+    Column {
+        CodePagerBar(selectedFileIndex, codePagerTitles, onFileClicked = {
+            selectedFileIndex = it
+        })
+        if (codePagerTitles[selectedFileIndex].contains(".md")) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
```

Comment:
This modifier should allow the entire screen to scroll and the changes from my PR (removing `TopAppBarScrollBehavior`) would only scroll this column rather than the whole screen. 
```suggestion
            Column(
                modifier = Modifier
                    .verticalScroll(rememberScrollState())
                    .weight(weight = 1f, fill = false),
                horizontalAlignment = Alignment.CenterHorizontally,
```